snapcraft/commands/status.py

```diff
@@ -419,3 +419,120 @@ class StoreTracksCommand(StoreListTracksCommand):
 
     name = "tracks"
     hidden = True
+
+
+class StoreListRevisionsCommand(BaseCommand):
+    """Command to list-revisions."""
+
+    name = "list-revisions"
+    help_msg = "List published revisions for <snap-name>"
+    overview = textwrap.dedent(
+        """
+        Examples:
+            snapcraft list-revisions my-snap
+            snapcraft list-revisions my-snap --arch armhf
+            snapcraft revisions my-snap
+        """
+    )
+
+    @overrides
+    def fill_parser(self, parser: "argparse.ArgumentParser") -> None:
+        parser.add_argument(
+            "snap_name",
+            metavar="snap-name",
+        )
+        parser.add_argument(
+            "--arch",
+            metavar="arch",
+            help="architecture filter",
```

Comment:
Most help messages start with uppercase, we can standardize on that.

Reply:
all the store ones are lower case, I created a task to consolidate